test/components/select/select-screen.test.js

```diff
@@ -331,13 +332,6 @@ describe("Select Screen", () => {
 			jest.spyOn(eventBus, "subscribe");
 		});
 
-		test("saves choice to transient data", () => {
-			mockChoices = [{ title: "Title 1" }];
-			selectScreen.create();
-
-			eventBus.subscribe.mock.calls[0][0].callback();
-			expect(selectScreen.transientData["test-select"].choice.title).toBe("Title 1");
-		});
```

Comment:
I think this might still be a valid test. Sorry, my fault 😂 

We do utilise 'selected' transientData for referencing the confirmed selection, i.e so the game knows what was selected:

https://github.com/bbc/genie/pull/532/files#diff-38a518a8d1ac1c089e299b6b631a4fe07a57a218dc99e27970da8800d1aa5d0bR102

Reply:
I've put it back 😄